packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -84,25 +77,14 @@ class CodeSizeBodyState extends State<CodeSizeBody>
     if (newController == controller) return;
     controller = newController;
 
-    snapshotRoot = controller.snapshotRoot.value;
-    addAutoDisposeListener(controller.snapshotRoot, () {
-      setState(() {
-        snapshotRoot = controller.snapshotRoot.value;
-      });
-    });
+    addAutoDisposeListener(controller.activeDiffTreeType);
 
-    diffRoot = controller.diffRoot.value;
-    addAutoDisposeListener(controller.diffRoot, () {
-      setState(() {
-        diffRoot = controller.diffRoot.value;
-      });
-    });
+    addAutoDisposeListener(controller.activeViewKey);
```

Comment:
We don't need to listen to this. The controller just needs to be updated on tapping to switch tabs.